src/Simulation/Simulation.jl

```diff
@@ -458,6 +456,57 @@ function _guess_optimal_number_of_threads_for_SOR(gs::NTuple{3, Integer}, max_nt
     return min(nextpow(2, max(cld(n+1, 25), 4)), max_nthreads)
 end
 
+"""
+    Grid(sim::Simulation{T}, contact_id::Int) where T
+
+Determines the grid for a certain contact, constrained by its symmetry.
+    
+## Arguments
+* `sim::Simulation{T}`: [`Simulation`](@ref) for which the grid should be determined.
+* `contact_id::Int` : Contact for which the symmetry (specific for this contact) constrained grid should be determined
+"""
+function Grid(sim::Simulation{T}, contact_id::Int) where T
```

Comment:
I am somewhat in favor of making `contact_id` a keyword argument...
From the outside, it is not easy to understand what `Grid(sim, 2)` might do..
Give you the second (?) `Grid` of the simulation?

Reply:
But then I can not use multiple dispatch anymore...